player.cpp

```diff
@@ -650,6 +650,47 @@ void Player::makeMove() {
       else
         getView()->presentText("Sorry", "Couldn't parse \"" + action.get<string>() + "\"");
       break;
+    case UserInputId::CHANGE_TRIBE: {
+      //Michal, is there a better way to do this?
```

Comment:
You could also enter an id number, create a TribeId with it (make the constructor public), and call Game::getTribe with it. Note that it will crash if the tribe doesn't exist, but you can change Game::getTribe to just return nullptr and check against that.